app/code/core/Mage/Adminhtml/Block/Widget/Form.php

```diff
@@ -192,7 +192,7 @@ protected function _setFieldset($attributes, $fieldset, $exclude = [])
                         'label'     => $attribute->getFrontend()->getLabel(),
                         'class'     => $attribute->getFrontend()->getClass(),
                         'required'  => $attribute->getIsRequired(),
-                        'note'      => $this->escapeHtml($attribute->getNote()),
+                        'note'      => $this->escapeHtml((string)$attribute->getNote()),
```

Comment:
```suggestion
                        'note'      => $this->escapeHtml($attribute->getNote()),
```
Revert after https://github.com/OpenMage/magento-lts/pull/2884